### ISSUE TITLE
‘Add section’ and ‘Add widget’ icon(+) buttons controls are not accessible using keyboard on ‘Home’ page.  

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#95]
User Experience:

If, any disable control get activated via keyboard, then the user who depends upon who depends upon keyboard/Screen Reader for navigation might get affected hence find it difficult to navigate properly on the expanded controls. 

 

Note: User credentials should NOT be included in the bug.

 
Repro-Steps

1. Open Azure portal with valid credentials.

2. Navigate to the any listed "API Management services" and select it.

3. Navigate to the "API" Option and select it.
4. Navigate to the "Echo API" button and select it.
5. Navigate to the various controls of operation tab of the "Design" pane like Front end, Inbound processing.
6. Navigate to the inbound process and select policy code editor button, select it.

7. Navigate to the insert policy button and expand it.

8. Navigate to the allow cross domain calls disable link and press enter key.

9. Verify, whether the link get activate via keyboard or not.

Actual Result:

On the API management page, disable "allow cross domain calls" link get activated via keyboard. Instead on pressing enter key control get collapse and visual focus appears on the editor content section.

 

Expected result:

On the API management page, disable "allow cross domain calls" link should not get activated via keyboard and the expanded control should be get collapse via pressing esc key.

Note:

Similar issue exist with Chrome.

Similar issue exist for set usage quota per subscription disable link.
 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
